Resources/Maps/_NF/Shuttles/osprey.yml

```diff
@@ -0,0 +1,6268 @@
+    - type: Transform
+      pos: -10.5,-0.5
+      parent: 1
+  - uid: 196
+    components:
+    - type: Transform
+      pos: -1.5,2.5
+      parent: 1
+  - uid: 197
+    components:
+    - type: Transform
+      pos: -0.5,3.5
+      parent: 1
+  - uid: 198
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 1.5,3.5
+      parent: 1
+  - uid: 199
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 2.5,2.5
+      parent: 1
+  - uid: 200
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 3.5,0.5
+      parent: 1
+  - uid: 201
+    components:
+    - type: Transform
+      pos: -2.5,0.5
+      parent: 1
+  - uid: 398
+    components:
+    - type: Transform
+      pos: 6.5,-1.5
+      parent: 1
+- proto: Gyroscope
+  entities:
+  - uid: 842
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: -3.5,-2.5
+      parent: 1
+  - uid: 903
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 4.5,-2.5
+      parent: 1
+- proto: LockerPilotFilled
+  entities:
+  - uid: 641
+    components:
+    - type: Transform
+      pos: 2.5,-0.5
+      parent: 1
+- proto: LockerQuarterMasterFilledHardsuit
+  entities:
+  - uid: 250
+    components:
+    - type: Transform
+      pos: -6.5,-4.5
+      parent: 1
+- proto: LockerSalvageSpecialistFilled
+  entities:
+  - uid: 678
+    components:
+    - type: Transform
+      pos: 11.5,-14.5
+      parent: 1
+  - uid: 679
+    components:
+    - type: Transform
+      pos: 11.5,-13.5
+      parent: 1
+  - uid: 680
+    components:
+    - type: Transform
+      pos: 11.5,-12.5
+      parent: 1
+- proto: LockerWallMaterialsFuelUraniumFilled
+  entities:
+  - uid: 262
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -11.5,-12.5
+      parent: 1
+- proto: LuxuryPen
+  entities:
+  - uid: 130
+    components:
+    - type: Transform
+      rot: 7.060250362421624 rad
+      pos: -0.484375,1.06875
+      parent: 1
+    - type: Physics
+      bodyStatus: InAir
+    - type: Fixtures
+      fixtures:
+        fix1:
+          shape: !type:PolygonShape
+            radius: 0.01
+            vertices:
+            - -0.25,-0.25
+            - 0.25,-0.25
+            - 0.25,0.25
+            - -0.25,0.25
+          mask:
+          - Impassable
+          - HighImpassable
+          layer: []
+          density: 20
+          hard: True
+          restitution: 0.3
+          friction: 0.2
+        throw-fixture:
+          shape: !type:PolygonShape
+            radius: 0.01
+            vertices:
+            - -0.25,-0.25
+            - 0.25,-0.25
+            - 0.25,0.25
+            - -0.25,0.25
+          mask:
+          - Impassable
+          - HighImpassable
+          - BulletImpassable
+          layer: []
+          density: 1
+          hard: False
+          restitution: 0
+          friction: 0.4
```

Comment:
Why is this pen getting yeeted?

Reply:
Well I threw it a ghost to get it lined up. When I buy the ship it seems fine...

Reply:
Map a fixed one, or remove the 20 lines of yaml issues from it.